cypress/e2e/requests/filter-request-by-pickup-service-point.cy.js

```diff
@@ -0,0 +1,67 @@
+import uuid from 'uuid';
+import testType from '../../support/dictionary/testTypes';
```

Comment:
will be better to import this from '../../support/dictionary' like: 
`import { TestTypes, DevTeams } from '../../support/dictionary';`

Reply:
done)